crates/api/src/site/purge/person.rs

```diff
@@ -28,18 +26,9 @@ pub async fn purge_person(
 ) -> Result<Json<SuccessResponse>, LemmyError> {
   // Only let admin purge an item
   is_admin(&local_user_view)?;
-
-  // Read the local user to get their images, and delete them
-  if let Ok(local_user) = LocalUserView::read_person(&mut context.pool(), data.person_id).await {
-    let pictrs_uploads =
-      LocalImage::get_all_by_local_user_id(&mut context.pool(), &local_user.local_user.id).await?;
-
-    for upload in pictrs_uploads {
-      delete_image_from_pictrs(&upload.pictrs_alias, &upload.pictrs_delete_token, &context)
-        .await
-        .ok();
-    }
-  }
+  delete_local_user_images(data.person_id, &context)
+    .await
+    .ok();
```

Comment:
Why dont you call `purge_user_account()` instead? Then there would also be no need for `delete_local_user_images()` as a separate function.

Reply:
Still unclear to me

Reply:
K done.